py/embedded-server/deephaven_server/server.py

```diff
@@ -120,22 +119,35 @@ def __init__(
         port: Optional[int] = None,
         jvm_args: Optional[List[str]] = None,
         extra_classpath: Optional[List[str]] = None,
+        default_jvm_args: Optional[List[str]] = None,
     ):
         """
         Creates a Deephaven embedded server. Only one instance can be created at this time.
+
+        Args:
+            host (Optional[str]): The host to bind the server to, defaults to None, meaning local host.
```

Comment:
This also doesn't default to "local host"; it defaults to the value specified in the configuration file, and otherwise falls back to "localhost".